pkg/app/piped/controller/scheduler.go

```diff
@@ -642,12 +643,14 @@ func (s *scheduler) reportMostRecentlySuccessfulDeployment(ctx context.Context)
 		retry = pipedservice.NewRetry(10)
 	)
 
+	// Update deployment status on remote.
```

Comment:
the called function name is self-explanatory so I don't think we need this comment.